messaging/src/main/java/org/axonframework/deadline/dbscheduler/DeadlineManagerNotSetException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.deadline.dbscheduler;
+
+import org.axonframework.common.AxonTransientException;
+
+/**
+ * Exception indicating a problem in the Deadline manager mechanism.
+ *
+ * @author Gerard Klijs
+ * @since 4.8.0
+ */
+public class DeadlineManagerNotSetException extends AxonTransientException {
```

Comment:
I'm not sure. I was considering the scenario where an application was still busy starting, and the `Scheduler` was already started, with a deadline task being executed, while the `DbSchedulerDeadlineManager` wasn't created yet. If there never were a `DbSchedulerDeadlineManager` to begin with, there would not be such a task. Hence, I think it's more likely to throw when it was not yet initialized instead of being a configuration error. We could also leave it in the middle and just extend from `AxonException`?

Reply:
Gotcha, that doesn't sound off at all. Let's make it an `AxonException` :+1: